daily_post.py

```diff
@@ -5,23 +5,88 @@
 import praw
 import requests
 import time
+import markovify
+from unidecode import unidecode
+import re
+import random
 
 
 from bot import *
 from bs4 import BeautifulSoup
 from jinja2 import Environment, FileSystemLoader
 
+
+STATE_SIZE = 2
+INFO_URL = 'https://github.com/trambelus/UserSim'
+SUB_URL = '/r/User_Simulator'
+SUB = 'SeattleWA'
+
+FOOTER = '\n\n-----\n\n[^^Info](%s) ^^| [^^Subreddit](%s)' % (INFO_URL, SUB_URL)
+
 reddit = praw.Reddit(client_id=CLIENT_ID,
                     client_secret=CLIENT_SECRET,
                     password=PASSWORD,
                     user_agent=USER_AGENT,
                     username=USERNAME)
 
-subreddit = reddit.subreddit('SeattleWA')
+subreddit = reddit.subreddit(SUB)
 
 now = datetime.datetime.now()
 
 
+class PText(markovify.Text):
+
+    def test_sentence_input(self, sentence):
+        emote_pat = re.compile(r"\[.+?\]\(\/.+?\)")
+        reject_pat = re.compile(r"(^')|('$)|\s'|'\s|([\"(\(\)\[\])])|(github.com/trambelus/UserSim)|(/r/User_Simulator)|(~\ [\w\d\-_]{3,20}\ -----)")
```

Comment:
Oh boy. haha 

Can you explain what you're trying to catch here? To me it looks like you're trying to filter out things in the Markov chain that are commonly posted?

Reply:
This was pulled from the user_simulator bot directly the comment they had on this was 
A basic sentence filter. This one rejects sentences that contain
		the type of punctuation that would look strange on its own
		in a randomly-generated sentence.

I think the github and /r/ portions are to prevent parsing their own footer again.